cmd/web/web.go

```diff
@@ -90,8 +116,23 @@ func serve(cmd *cobra.Command, args []string) {
 		log.Fatal("Failed to create the web application instance: ", err)
 	}
 
-	err = app.Start(ctx)
-	if err != nil {
+	if err := app.Start(ctx); err != nil {
 		log.Fatal("Error while running the web application server: ", err)
 	}
 }
+
+func prune(_ *cobra.Command, _ []string) {
+	olderThan := time.Duration(viper.GetInt("older-than")) * 24 * time.Hour
+
+	dbConfig := LoadDBConfig()
+	db, err := db.InitDB(dbConfig)
+	if err != nil {
+		log.Fatal("Error while initializing the database: ", err)
+	}
+
+	log.Infof("Pruning events older than %d days.", olderThan)
+	if err := datapipeline.PruneEvents(olderThan, db); err != nil {
+		log.Fatalf("Error while pruning older events: %s", err)
+	}
+	log.New().Infof("Events older than %d days pruned.", olderThan)
```

Comment:
Why do we have here this `log.New().Infof` instead of `log.Infof`?
Is there any difference?

Reply:
mmh i think it was copilot being to smart, good catch